core/ext/RibasimMakieExt.jl

```diff
@@ -0,0 +1,61 @@
+module RibasimMakieExt
+using Ribasim: Ribasim, Model
+if isdefined(Base, :get_extension)
+    using Makie
+    using DataFrames
+else
+    using ..Makie
+    using ..DataFrames
+end
```

Comment:
Our julia compat is such that get_extension will be defined, so no need for the else block.

Could you do

```
using Makie: a, b
using DataFrames: c, d
```

Such that we explicitly see what is being used?